### ISSUE TITLE
munge_sumstats.py column names specified on the command line should override any defaults

### ISSUE THREAD

**Issue by nshrine** [bulik/ldsc#272]
I specified on the command line to munge_sumstats.py

--snp ldsc_id

and yet it bombs out with the error:

ValueError: Found 2 different SNP columns

because there is another column called "rs" which I see is coded as a default possible SNP column name to look for.

The column name specified on the command line "ldsc_id" should override any defaults and this error should only occur when one doesn't specify a column name on the command line and multiple default column names match.

**Comment by ksenia007** [comment]
I agree this could be helpful. In the meantime, I think that using `--ignore` flag with the name of the column that can be interpreted as SNP will help with that error
